### DISCUSSION TITLE
Does the Spider V3.0 board have the 3.3v issue

### DISCUSSION THREAD

**Discussion by EKVoron24** [FYSETC/FYSETC-SPIDER#152] [Q&A] [answered]
I have recently purchased a Spider V3.0 board for a Voron2.4 printer and would like to know whether this board has the issue of burning the 3.3v on the board.

**Comment by cyberjak2k** [comment]
I've had the V3.0 since about mid last year and it's still going so I'd
have to guess no unless there's something specific that could cause it that
I have not done

**Reply by EKVoron24** [reply]
Thanks for the reply. 
Did you have to discharge the capacitors prior to plugging in the stepper drivers?

**Comment by cyberjak2k** [comment, marked as answer]
No I never have. But to be fair I didn't power the board up until all the
stepper drivers were plugged in

**Reply by EKVoron24** [reply]
Thanks again.
I'll put the stepper drivers in and then proceed with the rest of the set-up.
Regards
Eric
